README.md

```diff
@@ -130,15 +130,25 @@ They are always appended last to fzf's argument list. Because fzf uses the optio
 - [re-populate fzf's input list on demand](https://github.com/junegunn/fzf/issues/1750)
 - change the search mode
 
+### Change the command used to preview files
+
+The search directory feature, by default, uses `bat` to preview the contents of a file. `bat` is a well-adopted `cat` replacement with syntax highlighting, line numbers, and more. If you would like to change the preview tool (to `cat` for availability reasons, or to add custom logic like binary or image preview), you may set the `fzf_preview_file_cmd` variable. For example, in your `config.fish`, you may put:
+
+```fish
+set fzf_preview_file_cmd cat
+```
+
+Do not specify a target path in the command, as `fzf.fish` will [prepend the directory][custom preview command] to preview to the command itself.
```

Comment:
s/prepend the directory/prepend the file

Reply:
Whoops!